manufacturing/internals/display_panel/README.md

```diff
@@ -85,14 +73,34 @@ BEFORE purchasing any parts.**
 [a14key]:  https://www.digikey.com/en/products/detail/w-rth-elektronik/971090151/6174614
 [a15rw]:   electronics
 
+------
+## Custom Parts
+
+### Display Panel Acrylic Plate
+![](images/display_panel_acrylic_plate.jpg)
+
+| Download Type | Link   |
+|:--------------|:-------|
+| Inventor part file | [.ipt](display_panel_acrylic_plate.ipt) |
+| Drawing                    | [.pdf](display_panel_acrylic_plate.pdf) |
+| Export for laser cutting   | [.dxf](display_panel_acrylic_plate.dxf) |
+
+### Custom PCBs
+
+See **[Electronics Assembly](electronics)** for turnkey manufaturing files and full details.
+
+---------
 ## Assembly
 
+![](images/display_panel_assembly.jpg)
```

Comment:
So, wherever you have a thumbnail of the drawing, you might actually want to make it link to the PDF to open when you click on it, rather than just the JPG you are already showing.

Reply:
Interesting, If I can figure it out ill change this